library/cartridge_failover_promote.py

```diff
@@ -87,7 +89,11 @@ def failover_promote(params):
             msg="Failed to promote leaders: %s" % err
         )
 
-    return helpers.ModuleRes(changed=True)
+    new_active_leaders, _ = control_console.eval_res_err('''
```

Comment:
It's better to sort them before comparison

Reply:
It's a dict =).